feature/my/src/main/java/com/hankki/feature/my/MyJogboDetailRoute.kt

```diff
@@ -0,0 +1,227 @@
+import com.hankki.feature.my.component.StoreItem
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+
+@Composable
+fun MyJogboDetailRoute(
+    paddingValues: PaddingValues,
+    navigateUp: () -> Unit,
+    myJogboDetailViewMidel: MyJogboDetailViewModel = hiltViewModel()
+) {
+    val myJogboDetailState by myJogboDetailViewMidel.myJogboDetailState.collectAsStateWithLifecycle()
+
+    LaunchedEffect(true) {
+        myJogboDetailViewMidel.getMockStoreList()
+    }
+
+    MyJogboDetailScreen(
+        paddingValues = paddingValues,
+        navigateUp = navigateUp,
+        storeItems = myJogboDetailState.myStoreItems,
+    )
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun MyJogboDetailScreen(
+    paddingValues: PaddingValues,
+    navigateUp: () -> Unit,
+    storeItems: PersistentList<MyJogboDetailEntity>
+) {
+    val showDeleteDialog = remember { mutableStateOf(false) }
+    val showShareDialog = remember { mutableStateOf(false) }
+
+    if (showShareDialog.value) {
+        DialogWithDescription(
+            title = "등록된 식당이 있어요\n식당으로 이동할까요?",
+            description = "친구에게 내 족보를 공유할 수 있도록\n준비 중이에요",
+            buttonTitle = "확인",
+            onConfirmation = { showShareDialog.value = false }
+        )
+    }
+
+    if (showDeleteDialog.value) {
+        DialogWithButton(
+            onDismissRequest = { showDeleteDialog.value = false },
+            onConfirmation = { showDeleteDialog.value = false },
+            title = stringResource(R.string.delete_store),
+            textButtonTitle = stringResource(R.string.go_back),
+            buttonTitle = stringResource(id = R.string.delete)
+        )
+    }
+
+    Column(
+        modifier = Modifier
+            .padding(paddingValues)
+            .fillMaxSize()
+            .background(Red),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        HankkiTopBar(
+            modifier = Modifier.background(Red),
+            leadingIcon = {
+                Icon(
+                    painter = painterResource(id = com.hankki.core.designsystem.R.drawable.ic_arrow_left),
+                    contentDescription = "Back",
+                    modifier = Modifier
+                        .padding(start = 9.dp)
+                        .size(44.dp)
+                        .noRippleClickable(onClick = navigateUp)
+                )
+            },
+            content = {
+                Text(
+                    text = stringResource(R.string.my_store_jogbo),
+                    style = HankkiTheme.typography.sub3,
+                    color = Gray900
+                )
+            }
+        )
+
+        Spacer(
+            modifier = Modifier
+                .background(Red)
+                .height(4.dp)
+        )
+
+        JogboFolder(
+            title = storeItems[0].title,
+            chip1 = storeItems[0].tags[0],
+            chip2 = storeItems[0].tags[0],
```

Comment:
```suggestion
            chips = tags
```
chips를 list로 바꾸고 이렇게 넘기면 어떨까요